### DISCUSSION TITLE
Complete to fully qualified namespace for @var in doc block.

### DISCUSSION THREAD

**Discussion by sitedyno** [phpactor/phpactor#1893] [Q&A] [answered]
Is this possible? Completion using `cmp` inserts only the class name.

Or maybe some way to expand a class name to fully qualified namespace?

Thank you for all you do 🙇‍♂️ 

Edit: derp `:PhpactorClassExpand`

**Comment by dantleech** [comment, marked as answer]
Yes ClassExpand does this in PHP or any other type of file
